Cord blood mtd

drighelli opened the pull request:
adding metadata/colData for cord_blood dataset in CITEseq.

providing rda object through slack.

drighelli committed:
adding quality control masks to CITEseq cord_blood data

drighelli committed:
mostly updating documentation

LiNk-NY commented on R/CITEseq.R:
```diff
@@ -192,7 +215,9 @@ CITEseq <- function(DataType=c("cord_blood", "peripheral_blood"), modes="*",
             { stop("Unrecognized CITE-seq dataset name: ", DataType) }
         )
-        
+        if (filtered==TRUE) {
```
This should be `if (filtered)`

drighelli replied:
right! will be fixed in the next commit!

LiNk-NY commented:
Thank Dario @drighelli, it looks good. The merge should wait until we have the resources uploaded to `ExperimentHub`.

drighelli committed:
- adding cell filtering for ECCITEseq in dedicated script
- minor fixing in CITEseq

drighelli committed:
Merge pull request #5 from waldronlab/devel

adding rownames to ecciteseq

drighelli committed:
Merge branch 'cord_blood_mtd' of github.com:drighelli/SingleCellMultiModal into cord_blood_mtd

drighelli committed:
deleting a print in the code

drighelli committed:
- adding colData support to peripheral_blood (script+functions)
- minor bug fixing for peripheral_blood dataset
- converting SE in SCE in conversion function for CITEseq

drighelli committed:
Merge pull request #6 from waldronlab/devel

fixing ontomap

drighelli committed:
updating metadata file with peripheral_blood coldata

drighelli committed:
Merge branch 'cord_blood_mtd' of github.com:drighelli/SingleCellMultiModal into cord_blood_mtd

drighelli commented:
I can see that the latest commits on peripheral_blood dataset are already tracked in this PR.

LiNk-NY commented:
Hi Dario, @drighelli 
I don't see an entry in the metadata for the `peripheral_blood/v1.0.0/scRNAseq_coldata.rda`. Can you add this?

LiNk-NY reviewed (changes requested):
Please include an additional entry in the metadata for the `peripheral_blood` colData Rda file. Everything else looks good. Thanks

drighelli commented:
ahah sorry I edited the file without pushing it. 

check it now please! 

Thanks for the review!

LiNk-NY commented:
I'm still waiting on the updated EH database but the changes have been merged.
